vantage6/node/globals.py

```diff
@@ -30,10 +26,8 @@
 #    VPN CONFIGURATION RELATED CONSTANTS
 #
 # TODO move part of these constants elsewhere?! Or make context?
-VPN_CLIENT_IMAGE = 'algorithm-container-network_openvpn-client'
-NETWORK_CONFIG_IMAGE = 'network-config'
-VPN_SUBNET = '10.76.0.0/16'
-VPN_CONFIG_FILE = 'data/vpn-config.ovpn.conf'
-LOCAL_SUBNET_START = '172.'   # start of local IP addresses
+VPN_CLIENT_IMAGE = 'harbor2.vantage6.ai/infrastructure/vpn-client'
+NETWORK_CONFIG_IMAGE = 'harbor2.vantage6.ai/infrastructure/vpn-configurator'
```

Comment:
Remark: 
These need to be overwritable from the configuration file. I created a separate issue for this: https://github.com/IKNL/vantage6-node/issues/45

Lets do that in a patch or minor version